cut-n-paste/gimpcellrenderertoggle/gimpcellrenderertoggle.c

```diff
@@ -61,7 +61,11 @@ static void gimp_cell_renderer_toggle_set_property (GObject         *object,
                                                     GParamSpec      *pspec);
 static void gimp_cell_renderer_toggle_get_size     (GtkCellRenderer *cell,
                                                     GtkWidget       *widget,
+#if GTK_CHECK_VERSION (3, 0, 0)
+                                                    const GdkRectangle *rectangle,
+#else
                                                     GdkRectangle    *rectangle,
```

Comment:
gtk2 build don't give a warning here, so i make it gtk3 only.
Personal i don't want to touch gtk2 code if it isn't necessary or needed, on the last miles for migration to  gtk3.
Well, let us talk at irc about.